modules/admin_manual/pages/installation/manual_installation/upgrade_install_phpmyadmin.adoc

```diff
@@ -1,24 +1,24 @@
-= Upgrade `phpmyadmin`
+= Upgrade phpmyadmin
 :toc: right
 :toclevels: 2
 :phpmyadmin_home_url: https://www.phpmyadmin.net
 :phpmyadmin_dl_url: https://www.phpmyadmin.net/downloads/
 
+:description: This guide helps you to upgrade an existing and configured installation of {phpmyadmin_home_url}[phpmyadmin] from source.
+
 == Introduction
 
-This guide helps you to upgrade an existing installation of {phpmyadmin_home_url}[phpmyadmin]
-from source. This may be necessary if the provided version of your OS lags behind the available
-version and/or you see php errors when using it and you need an updated version.
+{description} This may be necessary if the provided version of your OS lags behind the available version and/or you see php errors when using it and you need an updated version.
```

Comment:
```suggestion
{description} This may be necessary if the provided version of your OS lags behind the available version and/or you see PHP errors when using it and you need an updated version.
```